### ISSUE TITLE
Koreader on Clara 2E randomly freezes

### ISSUE THREAD

**Issue by Emblaze1816** [koreader/koreader#10010]
Hello! 

I am experiencing a lot of problems with KOReader on my Clara 2E. Sometimes, while turning a page o selecting a word, it freezes and I can only hard reboot it or wait until it reboots itself. I tried converting this ebooks with Calibre but it continues to happen (maybe less often than before). I haven’t tried any other ebook because it happens only after a lot of pages and randomly.

The strangest thing is that that this issues don't happen regularly (i.e. today I have read form more than an hour and nothing happened; yesterday it happened two times). 

Some people [here on Reddit ](https://www.reddit.com/r/kobo/comments/106f4ql/koreader_on_my_clara_2e_randomly_freezes/)(I am the OP) say that happens to them too. 
Any advice? I am not a real IT expert, but I can say that KOReader is awesome and I am here to try to improve the experience for other users. 

Thank you

**Comment by Frenzie** [comment]
Duplicate of #9806. The summary is that it's the board or the kernel itself that has an issue. Current nightlies include some new attempts to work around those, please discuss in #9806.
